packages/just-stack-single-lib/template/tsconfig.json

```diff
@@ -4,10 +4,15 @@
     "module": "commonjs",
     "declaration": true,
     "declarationMap": true,
-    "outDir": "./lib",
```

Comment:
for single-libs, we're making node libraries - don't think that we would need to use lib-commonjs really

Reply:
nvm the task actually spits out both.